app/services/github_service.rb

```diff
@@ -9,4 +9,14 @@ def self.followers(current_user)
     response = Faraday.get("https://api.github.com/users/#{current_user.username}/followers?client_id=#{ENV['github_client_id']}&client_secret=#{ENV['github_client_secret']}")
     JSON.parse(response.body, symbolize_names: true)
   end
+
+  def self.following(current_user)
+    response = Faraday.get("https://api.github.com/users/#{current_user.username}/following?cliend_id=#{ENV['client_id']}&client_secret=#{ENV['client_secret']}")
```

Comment:
Take a look at the Faraday docs. You don't need to roll this URL by hand. You can set this as almost an attribute of the connection.